### PR TITLE
Enabled configuring ports and other cli args

### LINKED ISSUE #15: Picking Different Port Numbers
Looking through the code, it seems like only the default port numbers are allowed when starting 'siad'. There should be some option that allows you to start 'siad' on different ports. When setting up a developer network, each instance of siad needs its own port number for the api, the gateway rpcs, and the host rpcs.

### PR DESCRIPTION
**Pull request by Mingling94**
Fix #15

### CODE REVIEW

File: js/sia.js [hunk 1]
```diff
@@ -158,7 +160,13 @@ function SiadWrapper () {
...
     // Spawn siad
     const Process = require('child_process').spawn
-    var daemonProcess = new Process(siad.fileName, processOptions)
+    var daemonProcess = new Process(siad.fileName, {
+      '--agent': siad.agent,
+      '--api-addr': siad.address,
+      '--rpc-addr': siad.rpcAddress,
+      '--host-addr': siad.hostAddress,
+      '--sia-directory': siad.path
+    }, processOptions)
```
Comment (DavidVorick) [inline js/sia.js]: have you tested that this syntax is correct?

Comment (DavidVorick) [inline js/sia.js]: I wish there were unit tests here

Comment (Mingling94) [inline js/sia.js]: I've tested it just by running it with the UI, unit tests are a while away for the wrapper.

Comment (Mingling94) [inline js/sia.js]: JK, now it's correct ^_^

**Pull request merged**